### PR TITLE
Add CI tests for newer Benchmarks

### PR DESCRIPTION
**Pull request by aelligp**
This adds CI tests for the newer benchmarks such as:
- ShearBands2D
- Shearheating
- Sinking Block 2D
- etc. 

Addresses task 3 in #88

### CODE REVIEW

**Comment by albert-de-montserrat** [comment]
Remember to add JustPIC to `test/Project.toml`

**Comment by albert-de-montserrat** [comment]
And for those tests requiring time stepping, perhaps limit the number of time steps to a low number. Otherwise it will take ages to finish the CI

**Comment by aelligp** [comment]
In general they are quite quick, but I'll update them accordingly

**Comment by aelligp** [comment]
Tests pass in around 35min (roughly +10min from the previous CI testing).

Interestingly testsets (e.g. CellArrays2D) do not seem to count towards Codecov.

**Comment by albert-de-montserrat** [comment]
See this [PR](https://github.com/JuliaGeodynamics/JustPIC.jl/pull/63) related to the CodeCov

File: test/Project.toml [hunk 1]
```diff
@@ -1,7 +1,15 @@
...
 AllocCheck = "9b6a8646-10ed-4001-bbdc-1d2f46dfbb1a"
+JustPIC = "10dc771f-8528-4cd9-9d3b-b21b2e693339"
 ParallelStencil = "94395366-693c-11ea-3b26-d9b7aac5d958"
 StaticArrays = "90137ffa-7385-5640-81b9-e52037218182"
 Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
+Suppressor = "fd094767-a336-5f1f-9728-57cf17d0bbfb"
 TOML = "fa267f1f-6049-4f14-aa54-33bafae1ed76"
 Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
+
+[compat]
```
Comment (albert-de-montserrat) [inline test/Project.toml:11]: add a 0.2 lower bound compat for JustPIC

File: test/test_CellArrays2D.jl [hunk 2]
```diff
@@ -1,27 +1,27 @@
-using Test, StaticArrays, AllocCheck, JustRelax
+using Test, Suppressor, StaticArrays, AllocCheck, JustRelax
 using ParallelStencil
```
Comment (albert-de-montserrat) [inline test/test_CellArrays2D.jl:2]: I believe there is no need to load `ParallelStencil` here

File: test/test_CellArrays3D.jl [hunk 3]
```diff
@@ -1,25 +1,25 @@
 
-using Test, StaticArrays, AllocCheck, JustRelax
+using Test, Suppressor, StaticArrays, AllocCheck, JustRelax
 using ParallelStencil
```
Comment (albert-de-montserrat) [inline test/test_CellArrays3D.jl:3]: same as comment above

File: test/test_boundary_conditions2D.jl [hunk 4]
```diff
@@ -1,99 +1,101 @@
-using Test
+using Test, Suppressor
 using JustRelax
 using ParallelStencil
```
Comment (albert-de-montserrat) [inline test/test_boundary_conditions2D.jl:3]: no need to load

**Review by albert-de-montserrat — CHANGES_REQUESTED** [review]
Thanks a lot for the effort! looks good to me, just very few minor requests/comments

File: test/test_CellArrays2D.jl (hunk 2 above)
Comment (aelligp) [reply test/test_CellArrays2D.jl:2]: There is, tests fail due to PS not being initialised.

**Review by albert-de-montserrat — APPROVED** [review]

**Pull request merged**